habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -558,29 +558,40 @@ def action_space(self):
             }
         )
 
-    def step(self, human_joints_trans, is_last_action, **kwargs):
+    def step(self, *args, is_last_action, **kwargs):
```

Comment:
In the docstring, can you rename human_joints_trans to reflect that the actual argument is `self._action_arg_prefix + "human_joints_trans"`

Reply:
Addressed